src/components/AppsList/index.jsx

```diff
@@ -65,7 +72,12 @@ class AppsList extends Component {
 AppsList.propTypes = {
   apps: PropTypes.arrayOf(PropTypes.object),
   isRetrieved: PropTypes.bool,
```

Comment:
please explain these two props: `isRetrieving` and `isRetrieved`

Reply:
Well @ckwagaba:
- `isRetrieving` - is a boolean loading state (used to display Spinner) 
- `isRetrieved` - is another boolean state (used to indicate whether the loading is complete and successful)

@Mubangizi did this... he can confirm if I'm correct.

Reply:
Yeah, we use them for `spinner` and for `on failure` feedback, thats if is retrieved stays false